sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientLiveTests.cs

```diff
@@ -710,6 +710,67 @@ public async Task ProcessorClientDetectsAnInvalidStorageContainer(bool async)
             cancellationSource.Cancel();
         }
 
+        /// <summary>
+        ///   Verifies that the <see cref="EventProcessorClient" /> can stop when no events are
+        ///   available to read.
+        /// </summary>
+        ///
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public async Task ProcessorClientStopsWithoutWaitingForTimeoutWhenPartitionsAreEmpty(bool async)
```

Comment:
Do we already have a live test where we restart the processor after stopping? If not, is it worth adding one?

Reply:
Good thought; we don't have a Live test.  I'll add one.